diff --git a/node_modules/@thekonz/apollo-lighthouse-subscription-link/dist/index.es.js b/node_modules/@thekonz/apollo-lighthouse-subscription-link/dist/index.es.js
index 6c3d23c..8f3230e 100644
--- a/node_modules/@thekonz/apollo-lighthouse-subscription-link/dist/index.es.js
+++ b/node_modules/@thekonz/apollo-lighthouse-subscription-link/dist/index.es.js
@@ -3,7 +3,10 @@ import { ApolloLink, Observable } from '@apollo/client/core';
 function subscribeToEcho(echoClient, channelName, observer) {
     var channel = echoClient.private(channelName.replace(/^private\-/, ""));
     channel.listen(".lighthouse-subscription", function (result) {
-        return observer.next(result.data);
+        // Add result.result if result.data is null
+        // result.data is used by Echo protocol
+        // result.result is used by pusher protocol
+        return observer.next(result.data ?? result.result);
     });
 }
 function unsubscribe(echoClient, getChannelName) {
diff --git a/node_modules/@thekonz/apollo-lighthouse-subscription-link/dist/index.js b/node_modules/@thekonz/apollo-lighthouse-subscription-link/dist/index.js
index 944b5c8..5742925 100644
--- a/node_modules/@thekonz/apollo-lighthouse-subscription-link/dist/index.js
+++ b/node_modules/@thekonz/apollo-lighthouse-subscription-link/dist/index.js
@@ -7,7 +7,10 @@ var core = require('@apollo/client/core');
 function subscribeToEcho(echoClient, channelName, observer) {
     var channel = echoClient.private(channelName.replace(/^private\-/, ""));
     channel.listen(".lighthouse-subscription", function (result) {
-        return observer.next(result.data);
+        // Add result.result if result.data is null
+        // result.data is used by Echo protocol
+        // result.result is used by pusher protocol
+        return observer.next(result.data ?? result.result);
     });
 }
 function unsubscribe(echoClient, getChannelName) {
